crates/ark/src/lsp/completions/sources/composite/search_path.rs

```diff
@@ -53,7 +53,7 @@ fn completions_from_search_path(
     let mut completions = vec![];
 
     const R_CONTROL_FLOW_KEYWORDS: &[&str] = &[
-        "if", "else", "for", "in", "while", "repeat", "break", "next", "return", "function",
+        "if", "else", "for", "in", "while", "repeat", "break", "next",
```

Comment:
Do we need to exclude any of these actually? The shorter ones won't be triggered if they are below the character limit for completions but it presumably wouldn't hurt to include them.

Reply:
As we discussed in person, I suspect it's not good that we exclude these at all. I will open a new issue shortly with a systematic look at how reserved words currently fare in positron's R completions. And that will lead to some new proposals.